server/scripts/create_database.sql

```diff
@@ -1,4 +1,4 @@
-DROP TABLE  IF EXISTS  users, clients;
+DROP TABLE  IF EXISTS  users, clients, booking_status, residency_status, CASCADE;
```

Comment:
```suggestion
DROP TABLE  IF EXISTS  users, clients, booking_status, residency_status CASCADE;
```
There shouldn't be a comma before `CASCADE`. The comma makes postgres think CASCADE is a table name

Reply:
done, thanks for catching that!